types.go

```diff
@@ -18,296 +18,39 @@ package fission
 
 import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	apiv1 "k8s.io/client-go/pkg/api/v1"
+
+	fv1 "github.com/fission/fission/pkg/apis/fission.io/v1"
 )
 
 type (
-	//
```

Comment:
Why does all this have to be moved?  Is it possible to leave these here and use them in the Kubernetes types as fission.* ?